nncf/hw_configs/dla_int8bfp_subgraphs.txt

```diff
@@ -0,0 +1 @@
+(LINEAR_OPS | ARITHMETIC ) + (ANY_BN_ACT_COMBO | (ANY_BN_ACT_COMBO + max_pool2d)))
```

Comment:
For reasons of maintaining consistency, could I ask you to add the `cpu_subgraphs.txt`, `gpu_subgraphs.txt` and `vpu_subgraphs.txt`, all having the contents corresponding to the originally "default" pattern generated in code? Would like to make the default, code-generated subgraph path only trigger for unknown HW types, if at all.

Reply:
Sure - I'd omitted to minimise changes outside the scope of our work..